libr/core/cmd_anal.c

```diff
@@ -9383,7 +9397,12 @@ static int cmd_anal_all(RCore *core, const char *input) {
 		break;
 	case 'e': // "aae"
 		if (input[1] == 'f') { // "aaef
-			r_core_cmd0 (core, "aef@@@F");
```

Comment:
there's one test for @@@F, but maybe worth thinking in adding more tests for the rest of @@@ iterators (random though out of the scope of this PR)